modules/react/button/lib/TertiaryButton.tsx

```diff
@@ -41,18 +37,15 @@ export interface TertiaryButtonProps extends Themeable, BaseButtonProps {
    */
   /**
    * If set to `true`, transform the icon's x-axis to mirror the graphic
-   * @default false
    */
   shouldMirrorIcon?: boolean;
   /**
    * If set to `true`, transform text to all letters uppercase
-   * @default undefined
    */
   allCaps?: boolean;
   children?: React.ReactNode;
   /**
    * If set to `true`, make icon button available to use theme colors instead of default
-   * @default false
```

Comment:
The default for `isThemeable` isn't showing up either in TertiaryButton's props table.